src/pages/nfts/[...address].tsx

```diff
@@ -23,17 +24,17 @@ export async function getServerSideProps({ req, query }: NextPageContext) {
   const subdomain = req?.headers['x-holaplex-subdomain'];
   
   const {
-    data: { storefront, nft },
+    data: { marketplace, nft },
   } = await client.query<GetNftPage>({
     query: gql`
       query GetNftPage($subdomain: String!, $address: String!) {
-        storefront(subdomain: $subdomain) {
-          title
-          description
-          logoUrl
-          faviconUrl
-          bannerUrl
-          ownerAddress
+        marketplace(subdomain: $subdomain) {
+          subdomain,
+          name,
+          description,
+          logoUrl,
+          bannerUrl,
+          auctionHouseAddress
```

Comment:
Yep. This is totally correct. querying storefront was a temp measure until we had storefront v2 + auction house available in the index.